pyasdf/asdf_data_set.py

```diff
@@ -1464,6 +1464,200 @@ def split(container, count):
 
         return results
 
+    def process_two_files_with_parallel_output(
+            self, other_ds, process_function, output_filename,
+            traceback_limit=3):
```

Comment:
This method can basically be merged into `process_two_files_without_parallel_output` by checking argument `output_filename`